scripts/git-secrets-check.sh

```diff
@@ -15,9 +15,15 @@ echo "Git-Secrets Config loaded:"
 grep -A10 secrets .git/config
 # grep will return non-zero code if nothing found, failing the build
 
+PATH="$PATH:/tmp/git-secrets"
 echo "git-secrets-check.sh: Scanning repo ..."
 /tmp/git-secrets/git-secrets --scan -r ../
```

Comment:
If we can invoke it as `git secrets` vs `/tmp/git-secrets`, that would be preferable. Perhaps the `--install` arg can make this possible and eliminate the need for this PATH hijack.